### ISSUE TITLE
Bug_2016-07-11_Email Participant List

### ISSUE THREAD

**Issue by trungluongquang** [clemens-huebner/amos-ss16-proj8#189]
This function works on my iPhone.However, there is always a warning notification of Internet error before opening the email

**Comment by clemens-huebner** [comment]
@trungluongquang What do you mean with "before opening the email"?
You press the button in the app, your mail programm opens, and then comes the warning?

**Comment by trungluongquang** [comment]
@clemens-huebner yes. I can still send email but there is always a warning

**Comment by clemens-huebner** [comment]
@trungluongquang But the warning is from the external mail programm, right?

**Comment by clemens-huebner** [comment]
@trungluongquang So I don't think we can influence the external mail programm that much.
Do you get the same problem when writing support mail (it uses the same plugin)?

**Comment by trungluongquang** [comment]
no. The warning is from out apps. It appears before the email program runs.
The warning notification is the Internet error
